tests/test_init.py

```diff
@@ -129,26 +139,32 @@ def test_diff_output_for_diff_seed(self, init, sgntr, seed, tol):
         p1 = init(**s)
         s = {**s, 'seed': seed + 1}
         p2 = init(**s)
-        assert not np.allclose(p1, p2, atol=tol)
+
+        if p1.shape != (0,):
+            assert not np.allclose(p1, p2, atol=tol)
```

Comment:
I realised that zero is an edge case here, so added this if statement.

Reply:
Is this going to be an edge case for every template? If so, it could be organized into a separate test function. Not sure if it would make this file too big though. Anyways, porting the rest of the templates, in this case, would be out of scope and could be done in a subsequent PR.

Reply:
Good point, I'll mark it for another PR, this would make things very messy.